Automattic/jetpack
# AI Assistant: tweak assistant send button

Fixes https://github.com/Automattic/jetpack/issues/30835

## Proposed changes:

* AI Assistant: tweak assistant send button

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create an AI Assistant block instance
* Confirm the changes in the send button

<img width="702" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/a59a191e-9fc3-4930-9259-ec1b75c87725">

<img width="695" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/be4a9644-0445-4d0b-8815-7d72d00c84c8">

<img width="685" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/4c9e7ef5-802d-4261-82fb-b3fab5492e08">
